frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -12,7 +12,8 @@
 $co-affinity-row-margin: 15px;
 
 .co-catalog-tile {
-  height: 290px;
+  height: 100%;
```

Comment:
The cards now have different heights.
height is 203px
![Screen Shot 2020-02-04 at 2 03 49 PM](https://user-images.githubusercontent.com/895728/73777434-69bde180-4757-11ea-8047-80941f6917bb.png)
height is 224px
![Screen Shot 2020-02-04 at 2 03 59 PM](https://user-images.githubusercontent.com/895728/73777435-69bde180-4757-11ea-9f87-5d4443534e38.png)

Reply:
This was a recommendation from one of the PatternFly CSS devs. If we have fixed heights on these cards, we run into either ugly overflow bottom padding clipping or huge cards because we have to account for the largest possible card. The title and provider varies by as much as 1-5 lines that I've seen. With this method, the heights are consistent across the row and we avoid both of the fixed height issues.